react/src/App.tsx

```diff
@@ -133,9 +135,11 @@ function App() {
                 </Route>
               </Route>
             </Route>
+            <Route path="qrreader" element={<QrReaderView />} />
           </Route>
         </Route>
       </Route>
+      <Route path="/*" element={<NotFoundView />} />
```

Comment:
When using the route `/bases` only, this works (I suppose) by redirecting me to `/bases/1`. However when I go to `/bases/1/foo` I end up with `/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/bases/1/foo` so some recursion happens. Can you remove the usage of NotFoundView here because it's rather out-of-scope and fix it later?

Reply:
lol. ok yes. can do

Reply:
fixed it.